contessa/models.py

```diff
@@ -137,6 +156,73 @@ def __repr__(self):
         return f"Rule ({self.attribute} - {self.rule_name} - {self.task_ts})"
 
 
+class ConsistencyCheck(AbstractConcreteBase, DQBase):
+    """
+    Representation of abstract consistency check table.
+    """
+
+    __abstract__ = True
+
+    id = Column(BIGINT, primary_key=True)
+    type = Column(TEXT)
+    name = Column(TEXT)
+    description = Column(TEXT)
+    left_table = Column(TEXT)
+    right_table = Column(TEXT)
+
+    status = Column(TEXT)
+    time_filter = Column(TEXT)
+    task_ts = Column(TIMESTAMP(timezone=True), nullable=False, index=True)
+    created_at = Column(
+        DateTime(timezone=True),
+        server_default=text("NOW()"),
+        nullable=False,
+        index=True,
+    )
+
+    @declared_attr
+    def __table_args__(cls):
+        """
+        Concrete classes derived from this abstract one should have unique check among the columns
+        that below. But the constraint needs to have unique name, therefore we are using
+        @declared_attr here to construct name of the constraint using its table name.
+        :return:
+        """
+        return (
+            UniqueConstraint(
+                "type",
+                "name",
+                "left_table",
+                "right_table",
+                "task_ts",
+                "time_filter",
+                name=f"{cls.__tablename__}_unique_consistency_check",
+            ),
+        )
+
+    def init_row(
+        self,
+        check: Dict,
+        status: str,
+        left_table_name: str,
+        right_table_name: str,
+        context: Dict = None,
+    ):
+        """
+        Set result to consistency check object.
+        """
+        # todo - add to doc
+        self.task_ts = context["task_ts"]
+        self.name = check["name"]
```

Comment:
> Agreed, changed. I would even go further and put these basic attributes into a top level class in the future so that we dont have to watch consistency of names across all types of checks.

i don't know how to use this thing, but I can't comment on changed lines...

to comment - yea agreed, lets do it in another PR